providers/instantly.go

```diff
@@ -37,4 +42,41 @@ func init() {
 			Write:     true,
 		},
 	})
+
+	// Instantly v2 configuration
+	SetInfo(InstantlyAI, ProviderInfo{
+		DisplayName: "Instantly",
+		AuthType:    ApiKey,
+		BaseURL:     "https://api.instantly.ai/api",
+		ApiKeyOpts: &ApiKeyOpts{
+			AttachmentType: Header,
+			Header: &ApiKeyOptsHeader{
+				Name:        "Authorization",
+				ValuePrefix: "Bearer ",
+			},
+			DocsURL: "https://developer.instantly.ai/",
```

Comment:
DocsURL is supposed to be the doc link for how to get an API key, not just the main page of the developer docs. Please use https://developer.instantly.ai/getting-started/getting-started#generate-a-new-api-key